Bottom: 4dcbd8e547c863967eeb8678aca0d41859127772
Top:    8a1274c40c92b16743f39d781cac2905a3ed6771
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-15 18:39:38 +0000

find.h: Annotate which exported functions are not used!


---

diff --git a/src/find.h b/src/find.h
index d811634..4aa1e62 100644
--- a/src/find.h
+++ b/src/find.h
@@ -47,8 +47,8 @@ bool LineLineIntersect (LineType *, LineType *);
 bool LineArcIntersect (LineType *, ArcType *);
 bool PinLineIntersect (PinType *, LineType *);
 bool LinePadIntersect (LineType *, PadType *);
-bool ArcPadIntersect (ArcType *, PadType *);
-bool IsPolygonInPolygon (PolygonType *, PolygonType *);
+/* NOT USED EXTERNALLY */ bool ArcPadIntersect (ArcType *, PadType *);
+/* NOT USED EXTERNALLY */ bool IsPolygonInPolygon (PolygonType *, PolygonType *);
 void LookupElementConnections (ElementType *, FILE *);
 void LookupConnectionsToAllElements (FILE *);
 void LookupConnection (Coord, Coord, bool, Coord, int, bool AndRats);
@@ -57,17 +57,17 @@ bool ResetFoundLinesAndPolygons (bool, int flag);
 bool ResetFoundPinsViasAndPads (bool, int flag);
 bool ResetConnections (bool, int flag);
 void InitConnectionLookup (void);
-void InitComponentLookup (void);
-void InitLayoutLookup (void);
+/* NOT USED EXTERNALLY */ void InitComponentLookup (void);
+/* NOT USED EXTERNALLY */ void InitLayoutLookup (void);
 void FreeConnectionLookupMemory (void);
-void FreeComponentLookupMemory (void);
-void FreeLayoutLookupMemory (void);
+/* NOT USED EXTERNALLY */ void FreeComponentLookupMemory (void);
+/* NOT USED EXTERNALLY */ void FreeLayoutLookupMemory (void);
 void RatFindHook (int, void *, void *, void *, bool, bool);
 void SaveFindFlag (int);
 void RestoreFindFlag (void);
 int DRCAll (void);
-bool IsLineInPolygon (LineType *, PolygonType *);
-bool IsArcInPolygon (ArcType *, PolygonType *);
-bool IsPadInPolygon (PadType *, PolygonType *);
+/* NOT USED EXTERNALLY */ bool IsLineInPolygon (LineType *, PolygonType *);
+/* NOT USED EXTERNALLY */ bool IsArcInPolygon (ArcType *, PolygonType *);
+/* NOT USED EXTERNALLY */ bool IsPadInPolygon (PadType *, PolygonType *);
 
 #endif
